core/modules/ckeditor/ckeditor.info

```diff
@@ -9,3 +9,9 @@ version = BACKDROP_VERSION
 backdrop = 1.x
 
 configure = admin/config/content/formats
+
+; This module is deprecated, so hide it from new installs. Existing sites that
+; have CKEditor 4 module enabled will continue to see it in the modules list,
+; but once it is disabled, it will be hidden. Temporarily removing this line
+; will allow the module to be enabled if needed.
+hidden = TRUE
```

Comment:
TBH, I have concerns here. "Hidden" also means it's impossible to disable the module via admin UI, in case it's not needed anymore. It will stay enabled until B 2.x?
Doesn't seem ideal.

We also can't assume, all people switch to CKE5 - they might decide for a different editor, possibly TinyMCE. I think, people should have the ability to turn it off as desired. That won't work when the hidden key is present here.

Reply:
> It's claimed, that existing sites will still see it - that's not correct. It will be gone also for existing sites.

Then that sounds like a bug, because the way I understand this from previous discussions is that it should behave as described above.

Reply:
I might have misunderstood the way hidden works. Maybe that's the way it was/is in Drupal. I'll look into it.